include/seahorn/Expr/EvalUtils.hh

```diff
@@ -0,0 +1,43 @@
+
+#pragma once
+
+#include "seahorn/Expr/ExprCore.hh"
+#include "seahorn/Expr/ExprGmp.hh"
+#include "seahorn/Expr/ExprOpBind.hh"
+#include "seahorn/Expr/ExprOpBinder.hh"
+
+#include <math.h>
+#include <stdlib.h>
+
+namespace expr {
+namespace eval {
+namespace evalUtils {
+
+/// removes upper bits
+template <typename T> T overflow(const T &num, unsigned numBits) {
+  T mask = 0;
+
+  for (int i = 0; i < numBits; i++) {
+    mask = mask | ((((T)1) << i));
+  }
+
+  return num & mask;
+}
+
+/// removes lower bits
+template <typename T> T maskLowerBits(const T &num, unsigned numBits) {
```

Comment:
why is one function called `masked...` another is `overflow` but comment makes seem like they are related.
Maybe use related names as well?